notebooks/MUS2019_CTF.ipynb

```diff
@@ -0,0 +1,1120 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "MUS2019-CTF.ipynb",
+      "version": "0.3.2",
+      "provenance": [],
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python2",
+      "display_name": "Python 2"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/obsidianforensics/timesketch/blob/ctf-colab/notebooks/MUS2019_CTF.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
wondering whether this gets automatically changed once this gets merged... otherwise it is pointing to your branch

Reply:
Yeah I'm not sure - I can do another PR after the merge with the new one if it doesn't?

Reply:
I changed it to what I think the master path should be - I can fix afterwards if wrong.